### PR TITLE
Miner change to mention notary node dosen't need web server, and also change the total terminal windows number from 8 to 5

### PR DESCRIPTION
**Pull request by lijiachuan1982**
👮🏻👮🏻👮🏻 !!!! DESCRIBE YOUR CHANGES HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ Y] Have you run the unit, integration and smoke tests as described here? https://docs.corda.net/head/testing.html
- [Y ] If you added/changed public APIs, did you write/update the JavaDocs?
- [Y ] If the changes are of interest to application developers, have you added them to the changelog, and potentially release notes?
- [ Y] If you are contributing for the first time, please read the agreement in CONTRIBUTING.md now and add to this Pull Request that you agree to it.

Thanks for your code, it's appreciated! :)

### CODE REVIEW

File: docs/source/hello-world-state.rst [hunk 1]
```diff
@@ -24,17 +24,6 @@ interface is defined as follows:
             val participants: List<AbstractParty>
         }
 
-The first thing you'll probably notice about this interface declaration is that its not written in Java or another
```
Comment (joeldudleyr3) [inline docs/source/hello-world-state.rst]: Did you intend to remove this section?

File: docs/source/hello-world-running.rst [hunk 2]
```diff
@@ -70,7 +70,7 @@ the three node folders. Each node folder has the following structure:
 
         .
         |____corda.jar                     // The runnable node
-        |____corda-webserver.jar           // The node's webserver
+        |____corda-webserver.jar           // The node's webserver(Notary doesn't need a web server)
```
Comment (joeldudleyr3) [inline docs/source/hello-world-running.rst]: Can you add a space, and use lower-case 'n' for "Notary"? e.g. `The node's webserver (the notary doesn't need a web server)`.

**Comment by joeldudleyr3** [comment]
@lijiachuan1982 Would you also be able to state that your contribution is in accordance with the https://developercertificate.org/?

File: docs/source/hello-world-state.rst (hunk 1 above)
Comment (lijiachuan1982) [reply docs/source/hello-world-state.rst]: Hi @joeldudleyr3 , yes as this section is duplicated with the content which is under the "note", so I just removed this one and keep the later one in "note" part.

**Comment by lijiachuan1982** [comment]
my contribution is in accordance with the https://developercertificate.org

**Pull request merged**